projects/policekillings/pages/_app.js

```diff
@@ -0,0 +1,24 @@
+import Head from 'next/head'
+import { GlpThemeProvider } from 'components'
+
+import '../styles/globals.css'
+
+function MyApp({ Component, pageProps }) {
+  return (
+    <>
+      <Head>
+        <meta
+          name="viewport"
+          content="minimum-scale=1, initial-scale=1, width=device-width"
+        />
+        <title>Project One</title>
```

Comment:
Let's change 'Project One' here to match the police killings project as well. This is what will show up in the tab in your browser, but is also important for assistive technologies like screen readers to provide context.